src/Components/SharedComponents/EmptyAccount.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+    Title,
+    Button,
+    EmptyState,
+    EmptyStateIcon,
+    EmptyStateBody
+} from '@patternfly/react-core';
+import WrenchIcon from '@patternfly/react-icons/dist/esm/icons/wrench-icon';
+import { useIntl } from 'react-intl';
+import messages from '../../Messages';
+import propTypes from 'prop-types';
+
+const EmptyAccount = ({ message, className }) => {
+    const intl = useIntl();
+    return (
+        <EmptyState className={className} variant='large'>
+            <EmptyStateIcon icon={WrenchIcon} />
+            <Title headingLevel="h4" size="lg">
+                {intl.formatMessage(messages.emptyAccountTitle)}
+            </Title>
+            <EmptyStateBody>
+                {message}
+            </EmptyStateBody>
+            <Button
+                variant="primary"
+                component="a"
+                href={`https://access.redhat.com/documentation/en-us/red_hat_insights/2021/html/
```

Comment:
I guess we are wait for the actual URL from the doc team, and then we can merge this.